content/en/hosting/cht/migration/preparing-for-5.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "Preparing to upgrade to CHT 5.0"
+linkTitle: "Preparing for 5.0"
+weight: 
```

Comment:
suggestion: Let's move this guide before the "Preparing for 4.0" in the section hierarchy.

Reply:
good thinking! fixed in d6b12065 

turns out the helm chart migration doc didn't have frontmatter?  I added it tweaked the  title a bit